app/src/main/java/org/feature/fox/coffee_counter/ui/items/ItemsView.kt

```diff
@@ -2,41 +2,16 @@ package org.feature.fox.coffee_counter.ui.items
 
 import android.annotation.SuppressLint
 import androidx.compose.foundation.clickable
-import androidx.compose.foundation.layout.Arrangement
-import androidx.compose.foundation.layout.Box
-import androidx.compose.foundation.layout.Column
-import androidx.compose.foundation.layout.PaddingValues
-import androidx.compose.foundation.layout.Row
-import androidx.compose.foundation.layout.fillMaxSize
-import androidx.compose.foundation.layout.fillMaxWidth
-import androidx.compose.foundation.layout.height
-import androidx.compose.foundation.layout.padding
-import androidx.compose.foundation.layout.size
-import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.*
 import androidx.compose.foundation.rememberScrollState
 import androidx.compose.foundation.shape.CircleShape
 import androidx.compose.foundation.text.KeyboardOptions
 import androidx.compose.foundation.verticalScroll
-import androidx.compose.material.Button
-import androidx.compose.material.ButtonDefaults
-import androidx.compose.material.Divider
-import androidx.compose.material.ExtendedFloatingActionButton
-import androidx.compose.material.FloatingActionButton
-import androidx.compose.material.Icon
-import androidx.compose.material.MaterialTheme
-import androidx.compose.material.Scaffold
-import androidx.compose.material.Surface
-import androidx.compose.material.Text
-import androidx.compose.material.TextButton
+import androidx.compose.material.*
 import androidx.compose.material.icons.Icons
 import androidx.compose.material.icons.filled.Add
 import androidx.compose.material.icons.filled.Remove
-import androidx.compose.runtime.Composable
-import androidx.compose.runtime.getValue
-import androidx.compose.runtime.mutableStateOf
-import androidx.compose.runtime.remember
-import androidx.compose.runtime.rememberCoroutineScope
-import androidx.compose.runtime.setValue
+import androidx.compose.runtime.*
```

Comment:
We agreed on not using wildcard imports. 
You can change the settings under `Settings > Kotlin > Imports > Use single name import`

Reply:
Yes, sorry for that.

Reply:
No problem. You can fix this pretty easily if you run the autoformatter with `Optimize imports` enabled or right-click on a file and press `optimize imports`.

Reply:
Doing this automatically before each commit, but maybe had the wrong import settings

Reply:
Try to change them and adjust the wildcard-imports.

Reply:
Should be resolved with 2c08180d2d278185756874d42a79d0fb8c9638aa

Reply:
Should be resolved with 6b45923a02b9f4fe7deed5796c5e27f5d3696bf6